api/nakadi-event-bus-api.yaml

```diff
@@ -797,7 +797,74 @@ paths:
           description: Service (temporarily) unavailable
           schema:
             $ref: '#/definitions/Problem'
+    get:
```

Comment:
Do we want to have some query parameters for the first version for filtering of subscriptions during list-all operation? (Like "owning_application" or "event_types")